src/constants/chains.ts

```diff
@@ -50,5 +52,9 @@ export const RPC_URLS_MAP = process.env.ALCHEMY_API_KEY
         chain: bsc,
         url: `https://bnb-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`,
       },
+      [shape.id]: {
+        chain: shape,
+        url: `https://shape-mainnet.g.alchemy.com/v2/${process.env.ALCHEMY_API_KEY}`,
```

Comment:
assume you went ahead on turned on shape for this key, yea?

Reply:
yes